### PR TITLE
TW-120  Echo page refresh

### PR DESCRIPTION
**Pull request by jkonrath-postman**
* New screenshot
* Minor edits

### CODE REVIEW

File: src/pages/docs/developer/echo-api.md [hunk 1]
```diff
@@ -18,26 +18,26 @@ warning: false
```
Comment (jkonrath-postman) [inline src/pages/docs/developer/echo-api.md:18]: Add this line to the front matter at the top, after line 4:

`updated: 2021-12-06`

**Review by jkonrath-postman — COMMENTED** [review]
One little change, to add the `updated` date at top.

(I can't approve this because I initially created it. @mark-anthony-postman can, though.)

File: src/pages/docs/developer/echo-api.md [hunk 2]
```diff
@@ -18,26 +18,26 @@ warning: false
...
+To try the Echo API, open a new request in Postman and enter the following path in the URL field:
 
 ```http
 postman-echo.com/get
 ```
 
-Make sure `GET` is selected in the method drop-down, and click __Send__.
-
-[![Postman Echo response](https://assets.postman.com/postman-docs/postman-echo-api-response.jpg)](https://assets.postman.com/postman-docs/postman-echo-api-response.jpg)
+Make sure `GET` is selected in the method drop-down, then select __Send__. The Echo API will return a JSON object that contains details from the request.
```
Comment (mark-anthony-postman) [inline src/pages/docs/developer/echo-api.md:35]: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/dropdown
Per Microsoft, "dropdown" (no hyphen) should only be used as an adjective, so you could say "in the method dropdown list". Or you could simply avoid talking about UI items as Microsoft recommends and say something like "Make sure GET is selected for the method, then select..."

**Review by mark-anthony-postman — APPROVED** [review]
Made one minor comment, otherwise LGTM! 🙂

File: src/pages/docs/developer/echo-api.md (hunk 1 above)
Comment (HeatherStenson) [reply src/pages/docs/developer/echo-api.md:19]: Thanks @jkonrath-postman! Updated in ffa10d4

File: src/pages/docs/developer/echo-api.md (hunk 2 above)
Comment (HeatherStenson) [reply src/pages/docs/developer/echo-api.md]: Thanks @mark-anthony-postman, good call! Updated in f130af8

**Pull request merged**